discovery/scaleway/instance_test.go

```diff
@@ -122,7 +122,7 @@ func mockScalewayInstance(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "bad token id", http.StatusUnauthorized)
 		return
 	}
-	if r.RequestURI != "/instance/v1/zones/fr-par-1/servers?page=1" {
+	if r.URL.Path != "/instance/v1/zones/fr-par-1/servers" {
```

Comment:
I am also happy to check for page=1 from the query in a separate if statement if we think that is valuable.

Reply:
This is fine, we should focus on Prometheus testing, not underlying API SDK.